kubel.el

```diff
@@ -702,6 +706,20 @@ P can be a single number or a localhost:container port pair."
 						                              (kubel--get-resource-under-cursor)
 						                            (kubel--select-resource "Pods")))))
 
+(defun kubel-exec-shell-pod ()
+  "Exec into the pod under the cursor -> shell."
+  (interactive)
+  (kubel-setup-tramp)
+  (let* ((dir-prefix (or
+                    (when (tramp-tramp-file-p default-directory)
+                      (with-parsed-tramp-file-name default-directory nil
+                        (format "%s%s:%s@%s|" (or hop "") method user host))) ""))
+         (pod (if (kubel--is-pod-view)
+                  (kubel--get-resource-under-cursor)
+                (kubel--select-resource "Pods")))
+         (default-directory (format "/%skubectl:%s:/" dir-prefix pod)))
+    (shell (format "*kubel - shell - %s*" pod))))
```

Comment:
totally optional but do you know if there a way to use eshell or another terminal emulator here? it would be nice to have an option for users to set their preferred shell. But again, totally optional, I'm just curious.

Reply:
Ya, I should add `eshell` option. I will